DESCRIPTION

```diff
@@ -19,7 +19,7 @@ Authors@R: c(
 URL: https://github.com/olafmersmann/sendmailR
 BugReports: https://github.com/olafmersmann/sendmailR/issues
 Depends: R (>= 3.0.0)
-Imports: base64enc
+Imports: base64enc, curl (>= 4.0)
```

Comment:
Could we move `curl` to `Suggests` and conditionally load it in `.smtp_submit_mail_curl`? Then users won't encounter a new dependency when upgrading (or working in very restrictive environments)?

Reply:
curl is now optional.